src/portable/credentials.rs

```diff
@@ -4,14 +4,27 @@ use url::Url;
 use crate::options::Options;
 use crate::portable::options::ShowCredentials;
 
+#[derive(serde::Serialize)]
+struct Credentials {
+    #[serde(skip_serializing_if="Option::is_none")]
+    secret_key: Option<String>,
```

Comment:
This is something weird. Either we support secret_key in credentials JSON. Or we don't output it here. I think JSON output here should be readable as a credentials file.